admin_guide/managing_networking.adoc

```diff
@@ -1470,6 +1470,40 @@ to accept any connection allowed by each policy. That is,  connections on any
 port from pods in the *_same_* namespace, and connections on ports `80` and
 `443` from pods in *_any_* namespace.
 
+[[admin-guide-networking-networkpolicy-namespace-reduce-ovs-flow-rules]]
+=== Using Namespaces to reduce OVS Flow Rules generated by NetworkPolicy
```

Comment:
maybe something more generic about using NetworkPolicy efficiently, allowing for the possibility that we'll add other recommendations later?